habitat/omnitruck/hooks/init

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Removing previous version deployed at {{pkg.svc_static_path}}"
+rm -rf {{pkg.svc_static_path}}/* {{pkg.svc_static_path}}/.bundle
+
+echo "Deploying new version from {{pkg.path}} to {{pkg.svc_static_path}}"
+cp -a {{pkg.path}}/static/* {{pkg.path}}/static/.bundle {{pkg.svc_static_path}}
```

Comment:
Addressed this in my previous comment https://github.com/chef/omnitruck/pull/207#discussion_r99930061